pkg/kubelet/kubelet_node_status.go

```diff
@@ -315,7 +315,7 @@ func (kl *Kubelet) initialNode() (*v1.Node, error) {
 // It synchronizes node status to master, registering the kubelet first if
 // necessary.
 func (kl *Kubelet) syncNodeStatus() {
-	if kl.kubeClient == nil {
+	if kl.kubeClient == nil || kl.standaloneMode {
```

Comment:
Don't think this is needed. I think we should just remove the `standaloneMode` in the code and use `kubeClient` instead to simplify the logic.